### ISSUE TITLE
Unable to get the local Boot2Docker ISO version:  Did not find prefix "-v" in version string since v18.09

### ISSUE THREAD

**Issue by jiange17** [boot2docker/boot2docker#1362]
I am a newer for docker. When I execute the following statement.An infinite loop has occurred and I don't know how to solve it. please help me!

bogon: docker-machine create -d virtualbox mg
Running pre-create checks...
(mg) Default Boot2Docker ISO is out-of-date, downloading the latest release...
(mg) Latest release for github.com/boot2docker/boot2docker is v18.09.0
(mg) Downloading /Users/xiaochaoqun/.docker/machine/cache/boot2docker.iso from https://github.com/boot2docker/boot2docker/releases/download/v18.09.0/boot2docker.iso...
(mg) 0%....10%....20%....30%....40%....50%....60%....70%....80%....90%....100%
Creating machine...
(mg) Unable to get the local Boot2Docker ISO version:  Did not find prefix "-v" in version string
(mg) Default Boot2Docker ISO is out-of-date, downloading the latest release...
(mg) Latest release for github.com/boot2docker/boot2docker is v18.09.0
(mg) Downloading /Users/xiaochaoqun/.docker/machine/cache/boot2docker.iso from https://github.com/boot2docker/boot2docker/releases/download/v18.09.0/boot2docker.iso...

some info:

docker-machine -v
docker-machine version 0.16.0, build 702c267f

docker version
Client: Docker Engine - Community
 Version:           18.09.0
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        4d60db4
 Built:             Wed Nov  7 00:47:43 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.0
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       4d60db4
  Built:            Wed Nov  7 00:55:00 2018
  OS/Arch:          linux/amd64
  Experimental:     false

**Comment by alipig** [comment]
set the system variable VIRTUALBOX_BOOT2DOCKER_URL to redirect the download url.
you can use the http file server just as hfs.exe on http://www.rejetto.com/hfs/ to serve the iso image
good job

**Comment by jiange17** [comment]
> set the system variable VIRTUALBOX_BOOT2DOCKER_URL to redirect the download url.
> you can use the http file server just as hfs.exe on http://www.rejetto.com/hfs/ to serve the iso image
> good job

I am sorry, I did not explain in advance that I am using an Apple computer. In addition, my problem is that I successfully downloaded the iso image, but still "Unable to get the local Boot2Docker ISO version: Did not find prefix "-v" in version string", so it goes to get the latest version again. . When it is downloaded successfully, it will repeat the operation just now. In fact, I have downloaded the latest version. Maybe your answer is correct, but I don't understand it.
